django/tox.ini

```diff
@@ -18,6 +18,7 @@ passenv =
     RDWATCH_DJANGO_DEBUG
     RDWATCH_SMART_STAC_URL
     RDWATCH_SMART_STAC_KEY
+    RDWATCH_POSTGRESQL_SCORING_URI
```

Comment:
Also, I don't think we need to include this here anymore now that the `DATABASES` setting is a property and can handle `RDWATCH_POSTGRESQL_SCORING_URI` not being set. In fact, doing so causes tests to pass again for me, at least locally.
```suggestion
```

Reply:
Thanks for finding that.  Yeah, that must make only the default DB load so the tests no longer have an issue determining which DB to run on.  I looked up and tried some settings to set them manually but it didn't seem to work properly.